src/main/java/openfasttrack/cli/CommandLineInterpreter.java

```diff
@@ -116,10 +121,30 @@ public void parse()
         }
     }
 
+    private void handleSingleCharacterArgument(final ListIterator<String> iterator,
+            final String argument)
+    {
+        final String arguments = argument.replaceFirst(SINGLE_CHAR_ARG_PREFIX, "").toLowerCase();
+
+        int position = 0;
+        for (final String character : arguments.split(""))
```

Comment:
This splits a string into chars? This is not obvious. Maybe extract this to method `splitIntoCharacters()`? Or use [String.charAt(i)](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#charAt-int-) as you need the position counter anyway?